src/test/java/lotto/step2/domain/Fixture.java

```diff
@@ -0,0 +1,13 @@
+package lotto.step2.domain;
+
+import java.util.List;
+import java.util.Set;
+
+public class Fixture {
+    public static Lotto l1 = new Lotto(List.of(1, 2, 3, 4, 5, 6));
```

Comment:
```suggestion
    public static Lotto l1 = TestLotto.of(1, 2, 3, 4, 5, 6);

    static class TestLotto extends Lotto {

        public TestLotto(List<Integer> lotto) {
            super(lotto);
        }

        public static Lotto of(Integer... numbers) {
            return new Lotto(Arrays.stream(numbers).collect(Collectors.toList()));
        }
    }
```

- 생성로직이 복잡해질 경우, 메소드/클래스 분리 등을 통해 테스트코드 상에 중복을 없애는 것도 좋을거 같아요

Reply:
가변인자(...) 지정 사용해주신것 좋네요 ㅎㅎ
감사합니다!